include/ensmallen_bits/moead/moead_impl.hpp

```diff
@@ -300,7 +300,6 @@ Optimize(std::tuple<ArbitraryFunctionType...>& objectives,
         arma::conv_to<arma::mat>::from(population[solutionIdx]);
   }
 
-  EvaluateObjectives(population, objectives, populationFitness);
   // Set the candidates from the Pareto Front as the output.
   paretoFront.resize(populationFitness[0].n_rows, populationFitness[0].n_cols,
```

Comment:
Using [.set_size()](http://arma.sourceforge.net/docs.html#set_size) would be more efficient than [.resize()](http://arma.sourceforge.net/docs.html#resize). The latter preserves data, which doesn't look necessary here. All the slices of `paretoFront` are overwritten afterwards, so there doesn't seem anything to preserve.

Reply:
Nice catch!